
From: Miklos Szeredi <miklos@szeredi.hu>

This patch adds FUSE filesystem to MAINTAINERS, fs/Kconfig and
fs/Makefile.

Signed-off-by: Miklos Szeredi <miklos@szeredi.hu>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/MAINTAINERS |    7 +++++++
 25-akpm/fs/Kconfig  |   13 +++++++++++++
 25-akpm/fs/Makefile |    1 +
 3 files changed, 21 insertions(+)

diff -puN fs/Kconfig~fuse-maintainers-kconfig-and-makefile-changes fs/Kconfig
--- 25/fs/Kconfig~fuse-maintainers-kconfig-and-makefile-changes	Wed Jan 12 14:33:22 2005
+++ 25-akpm/fs/Kconfig	Wed Jan 12 14:33:22 2005
@@ -465,6 +465,19 @@ config CACHEFS
 
 endmenu
 
+config FUSE_FS
+	tristate "Filesystem in Userspace support"
+	help
+	  With FUSE it is possible to implement a fully functional filesystem
+	  in a userspace program.
+
+	  There's also companion library: libfuse.  This library along with
+	  utilities is available from the FUSE homepage:
+	  <http://fuse.sourceforge.net/>
+
+	  If you want to develop a userspace FS, or if you want to use
+	  a filesystem based on FUSE, answer Y or M.
+
 menu "CD-ROM/DVD Filesystems"
 
 config ISO9660_FS
diff -puN fs/Makefile~fuse-maintainers-kconfig-and-makefile-changes fs/Makefile
--- 25/fs/Makefile~fuse-maintainers-kconfig-and-makefile-changes	Wed Jan 12 14:33:22 2005
+++ 25-akpm/fs/Makefile	Wed Jan 12 14:33:22 2005
@@ -88,6 +88,7 @@ obj-$(CONFIG_QNX4FS_FS)		+= qnx4/
 obj-$(CONFIG_AUTOFS_FS)		+= autofs/
 obj-$(CONFIG_AUTOFS4_FS)	+= autofs4/
 obj-$(CONFIG_ADFS_FS)		+= adfs/
+obj-$(CONFIG_FUSE_FS)		+= fuse/
 obj-$(CONFIG_UDF_FS)		+= udf/
 obj-$(CONFIG_SUN_OPENPROMFS)	+= openpromfs/
 obj-$(CONFIG_JFS_FS)		+= jfs/
diff -puN MAINTAINERS~fuse-maintainers-kconfig-and-makefile-changes MAINTAINERS
--- 25/MAINTAINERS~fuse-maintainers-kconfig-and-makefile-changes	Wed Jan 12 14:33:22 2005
+++ 25-akpm/MAINTAINERS	Wed Jan 12 14:33:22 2005
@@ -897,6 +897,13 @@ L:	linux-tape@vger.kernel.org
 W:	http://sourceforge.net/projects/ftape
 S:	Orphan
 
+FUSE: FILESYSTEM IN USERSPACE
+P:	Miklos Szeredi
+M:	miklos@szeredi.hu
+L:	fuse-devel@lists.sourceforge.net
+W:	http://fuse.sourceforge.net/
+S:	Maintained
+
 FUTURE DOMAIN TMC-16x0 SCSI DRIVER (16-bit)
 P:	Rik Faith
 M:	faith@cs.unc.edu
_
